src/plugins/raven/sagas/connectionFlow.js

```diff
@@ -17,6 +17,7 @@ const maxConnectionAttempts = 5;
  * @param {Raven} raven
  */
 function* connect(raven) {
+  const maxConnectionAttempts = 5;
   let error;
   for (let i = 0; i < maxConnectionAttempts; i += 1) {
```

Comment:
@tbone1020 you can just hardcode this this to 5 like it was before since we wont be changing it